SoftLayer/managers/network.py

```diff
@@ -866,3 +866,19 @@ def get_datacenter_by_keyname(self, keyname=None):
         if len(result) >= 1:
             return result[0]
         return {}
+
+    def get_pods_with_capabilities(self):
```

Comment:
this method is same a this [def get_pods(self, datacenter=None)](https://github.com/softlayer/softlayer-python/blob/master/SoftLayer/managers/network.py#L789), I think the better way is refactor this method to work with filters and mask.

Reply:
Done